plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/Xenserver625StorageProcessor.java

```diff
@@ -544,6 +546,9 @@ public Answer backupSnapshot(final CopyCommand cmd) {
             } else {
                 newSnapshot.setParentSnapshotPath(prevBackupUuid);
             }
+            s_logger.info("##### New snapshot details: " + newSnapshot.toString());
+            s_logger.info("##### New snapshot physical utilization: "+physicalSize);
+
```

Comment:
Why the #### in the log line? We don't do that anywhere? Also, can't this be one line?